src/pages/sweepstake/mysweepstakes/index.jsx

```diff
@@ -0,0 +1,412 @@
+
+                try{
+                    let token = document.getElementById('selectToken').value
+                    let inputTokenURI = document.getElementById('inputTokenURI').value
+                    let duration = Number(document.getElementById('inputDuration').value)
+                    
+                    token == 'pBRL' ? token = process.env.NEXT_PUBLIC_CBRL :
+                    token == 'pUSD' ? token = process.env.NEXT_PUBLIC_CUSD :
+                    token == 'pEUR' ? token = process.env.NEXT_PUBLIC_CEUR :
+                    null                        
+                    
+                    if(token == null) return 
+
+                    await performActions(async (kit) => {
+                        
+                        Swal.close()
+
+                        let gasLimit = await contractERC721EsolidarSweepstake.methods
+                        .mint(inputTokenURI, token, duration * 60)
+                        .estimateGas()    
+
+                        await contractERC721EsolidarSweepstake.methods
+                        .mint(inputTokenURI, token, duration * 60)
+                        .send({ from: address, gasLimit })
+                        
+                        sweetAlertSuccess('Your campaign was created successfully!')
+
+                        getAllSweepstakesContract()
+                      })
+                                    
+                }catch(e){
+                    if(e?.reason?.includes('0x9f2df0fed2c77648de5860a4cc508cd0818c85b8b8a1ab4ceeef8d981c8956a6')){
+                        sweetAlertError('You need to be a Charity to create a campaing')
+                    }           
+                }
+            }
+          })
+    }
+    
+    function CountdownTimer(date, renderer) {
+        return (
+          <>
+            <Countdown
+              autoStart={true}
+              date={date}
+              renderer={({ days, hours, minutes, seconds, completed }) => {
+                if (completed) {
+                  return 'Completed'
+                } else {              
+                    if(days > 1 && hours > 1){
+                      return (
+                    <span>
+                      {days}d {hours}h
+                    </span>
+                      )
+                    }else{
+                      return (
+                        <span className="">
+                          {minutes}m {seconds}s
+                        </span>
+                      )
+                    }              
+                }
+              }}
+              />
+          </>
+    )};
+
+    async function cancelCampaing(nftID){
+        try{
+            await performActions(async (kit) => {
+      
+              let gasLimit = await contractERC721EsolidarSweepstake.methods
+              .burn(nftID)
+              .estimateGas()
+              
+              let res = await contractERC721EsolidarSweepstake.methods
```

Comment:
Codacy has a fix for the issue: ['res' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/esolidar/web3/pullRequest?prid=9542591)
```suggestion
              const res = await contractERC721EsolidarSweepstake.methods
```